.travis.yml

```diff
@@ -1,13 +1,20 @@
 language: python
+sudo: false
 python:
-    - 2.6
     - 2.7
-    - 3.3
     - 3.4
+    - 3.5
+    - 3.6
+    - pypy
+    - pypy3
 install:
-    - python bootstrap.py
-    - bin/buildout
+    - pip install -U pip setuptools
+    - pip install -U zope.testrunner coverage coveralls
+    - pip install -U -e .[test]
 script:
-    - bin/test -v1
+    - coverage run -m zope.testrunner --test-path=src
+after_success:
+    - coveralls
```

Comment:
How about something like `pip install coveralls && coveralls` here, and remove `coveralls` from `install:`?

That way, if a build fails, there's no need to install unnecessary dependencies -> faster builds + fail fast.

Reply:
Interesting idea, lets see if it can buy us something.

Reply:
@hugovk Maybe this is not such a good idea: The build cache is created before running the `after_success` section so `coveralls` and its dependencies do not make it into this cache. I fear this makes successful builds some seconds slower. Maybe this does not matter, so I'll keep the change for now.